src/Components/FilterNavbar/FilterNavbar.jsx

```diff
@@ -28,12 +33,12 @@ export function FilterNavbar() {
                         <div className="nav-sublists">
                             <ul>
                                 <li>
-                                    <input type="radio" name="c1" id="c1" checked = {initialState.sortBy && initialState.sortBy === "LOW_TO_HIGH"} onClick={() => filterAction({ type : "SORT_PRICE", payload : "LOW_TO_HIGH"})}/>
-                                    <label for="s1">High to Low</label>
+                                    <input type="radio" name="c1" id="c1" checked = {sortBy && sortBy === "LOW_TO_HIGH"} onClick={() => filterAction({ type : "SORT_PRICE", payload : "LOW_TO_HIGH"})}/>
+                                    <label htmlFor="s1">High to Low</label>
```

Comment:
You will have to change the id here to `s1`, so that when you click on the label the radio button gets selected.